src/Symfony/Bundle/FrameworkBundle/Resources/config/http_client.xml

```diff
@@ -22,5 +22,11 @@
             <argument type="service" id="Psr\Http\Message\StreamFactoryInterface" on-invalid="ignore" />
         </service>
         <service id="Psr\Http\Client\ClientInterface" alias="psr18.http_client" />
+
+        <service id="Http\Client\HttpClient" class="Symfony\Component\HttpClient\HttplugClient">
```

Comment:
not an alias so that it's not listed in debug:autowiring - no need to promote the alias for autowiring